modules/layers/src/column-layer/column-layer.js

```diff
@@ -103,24 +125,34 @@ export default class ColumnLayer extends Layer {
     }
   }
 
-  getGeometry(diskResolution) {
-    return new CylinderGeometry({
-      radius: 1,
-      topCap: false,
-      bottomCap: true,
-      height: 2,
-      verticalAxis: 'z',
-      nradial: diskResolution,
-      nvertical: 1
-    });
+  setGeometryState(gl, diskResolution) {
```

Comment:
The current way the geometry is calculated/overridden/updated looks a little messy. Not sure if there is a gotcha, but the way I would want to "clean up" this code would be to create a custom `Geometry`:

```
class ColumnGeometry extends CylinderGeometry {
  constructor(props) {
    super(props) {
    // post process positions here
}
```
I suppose we would regenerated the NORMAL/TEXCOORD_0 buffers unnecessarily but I think it would be well worth the readability win. And if that is a concern, we could add a method to our custom geometry that just mutates that attribute.

Then to update I would simply use 
```
model.setProps({
  geometry: new ColumnGeometry()
})
```

This wold ensure that I did not have to import `getBuffersFromGeometry`.